utils/filters.py

```diff
@@ -0,0 +1,80 @@
+import django
+from django.db.models import Value
+from django.db.models.functions import Lower, StrIndex
+import django_filters
+import graphene
+from graphene_django.forms.converter import convert_form_field
+from functools import partial
+
+
```

Comment:
Let's create a directory util/graphene/ and put all graphene related files there.